site/docs/status.md

```diff
@@ -0,0 +1,362 @@
+
+| Language  | Version |
+|-----------|---------|
+| Java      | 1.7.1   |
+| PyIceberg | 0.7.0   |
+| Rust      | 0.3.0   |
+| Go        | 0.1.0   |
+
+## Data Types
+
+| Data Type      | Java | PyIceberg | Rust | Go |
+|----------------|------|-----------|------|----|
+| boolean        | Y    | Y         | Y    | Y  |
+| int            | Y    | Y         | Y    | Y  |
+| long           | Y    | Y         | Y    | Y  |
+| float          | Y    | Y         | Y    | Y  |
+| double         | Y    | Y         | Y    | Y  |
+| decimal        | Y    | Y         | Y    | Y  |
+| date           | Y    | Y         | Y    | Y  |
+| time           | Y    | Y         | Y    | Y  |
+| timestamp      | Y    | Y         | Y    | Y  |
+| timestamptz    | Y    | Y         | Y    | Y  |
+| timestamp_ns   | Y    | Y         | Y    | Y  |
+| timestamptz_ns | Y    | Y         | Y    | Y  |
+| string         | Y    | Y         | Y    | Y  |
+| uuid           | Y    | Y         | Y    | Y  |
+| fixed          | Y    | Y         | Y    | Y  |
+| binary         | Y    | Y         | Y    | Y  |
+
+## Data File Formats
+
+| Format  | Java | PyIceberg | Rust | Go |
+|---------|------|-----------|------|----|
+| Parquet | Y    | Y         | Y    | Y  |
+| ORC     | Y    | N         | N    | N  |
+| Puffin  | Y    | N         | N    | N  |
+
+## File IO
+
+| Storage              | Java | PyIceberg | Rust | Go |
+|----------------------|------|-----------|------|----|
+| Local Filesystem     | Y    | Y         | Y    | Y  |
+| Hadoop Filesystem    | Y    | Y         | Y    | Y  |
+| Aws S3               | Y    | Y         | Y    | Y  |
+| Google Cloud Storage | Y    | Y         | Y    | Y  |
+| Memory Fs            | Y    | Y         | Y    | Y  |
+
+## Table Update Operations
+
+### Table Spec V1
+
+| Operation                   | Java | PyIceberg | Rust | Go |
+|-----------------------------|------|-----------|------|----|
+| Update schema               | Y    | N         | Y    | N  |
+| Update partition spec       | Y    | N         | Y    | N  |
+| Update table properties     | Y    | Y         | Y    | N  |
+| Replace sort order          | Y    | N         | N    | N  |
+| Update table location       | Y    | N         | N    | N  |
+| Append data files           | Y    | Y         | N    | N  |
+| Rewrite files               | Y    | Y         | N    | N  |
+| Rewrite manifests           | Y    | Y         | N    | N  |
+| Overwrite files             | Y    | Y         | N    | N  |
+| Row delta                   | Y    | N         | N    | N  |
+| Delete files                | Y    | N         | N    | N  |
+| Update statistics           | Y    | N         | N    | N  |
+| Update partition statistics | Y    | N         | N    | N  |
+| Expire snapshots            | Y    | N         | N    | N  |
+| Manage snapshots            | Y    | N         | N    | N  |
+
+### Table Spec V2
+
```

Comment:
I think users typically think of the Table CRUD operations separately from the Table Maintenance operations. Would it be helpful to separate these out into different categories?

Reply:
Yes, that's what I meant 💯 

I think this could be seen similar to what @Fokko [suggested above](https://github.com/apache/iceberg/pull/11772#pullrequestreview-2501651835) as well

Reply:
+1 to splitting